INTEGRATE.md

```diff
@@ -0,0 +1,31 @@
+# Local integration between Porta/System with Keycloak through Zync
+Install these 3 components to have a complete integration of managed SSO applications in Porta.
+
+## Install Porta
+Follow [the installation guide](https://github.com/3scale/porta/blob/master/INSTALL.md).
+
+## Install Zync
+Follow [the installation guide](https://github.com/3scale/zync/blob/master/INSTALL.md).
+
+## Install Keycloak
+To have a keycloak running locally, do `docker run --name keycloak -d -p 8080:8080 -e KEYCLOAK_USER=admin -e KEYCLOAK_PASSWORD=p jboss/keycloak`
+
+Finally [configure Red Hat Single Sign-On](https://access.redhat.com/documentation/en-us/red_hat_3scale_api_management/2.4/html/api_authentication/openid-connect).
+
+## Run Zync
+From the folder where Zync is installed, run: `ZYNC_AUTHENTICATION_TOKEN=zynctoken bundle exec rails server -p 5000` to get zync running in the port 5000.
+
+## Run Porta
+These configurations are needed:
+  - In `config/rolling_updates.yml`, make sure to have `apicast_v2: true` and `apicast_oidc: true`.
+  - In `config/sandbox_proxy.yml`, make sure to have `apicast_registry_url: <%= ENV.fetch('APICAST_REGISTRY_URL', '<Insert a URL that contains policies in JSON. It can be am empty JSON>') %>`
+
+Run porta with `UNICORN_WORKERS=8 ZYNC_ENDPOINT=http://localhost:5000 ZYNC_AUTHENTICATION_TOKEN=zynctoken bundle exec rails server -b 0.0.0.0`
+
+### Run Porta's sidekiq
+It works in Background in the `low` queue so do this: `ZYNC_ENDPOINT=http://localhost:5000 ZYNC_AUTHENTICATION_TOKEN=zynctoken bundle exec sidekiq -q default -q mailers -q low -q critical`
```

Comment:
Sidekiq Zync Worker process jobs from the `low` queue.

Reply:
@hallelujah What about this then?
```suggestion
Sidekiq Zync Worker process jobs from the `low` queue so do this: `ZYNC_ENDPOINT=http://localhost:5000 ZYNC_AUTHENTICATION_TOKEN=zynctoken bundle exec sidekiq -q low`
```